server/broadcast_test.go

```diff
@@ -101,7 +101,7 @@ func (s *stubOSSession) SaveData(name string, data []byte) (string, error) {
 func (s *stubOSSession) EndSession() {
 }
 func (s *stubOSSession) GetInfo() *net.OSInfo {
-	return nil
+	return &net.OSInfo{}
```

Comment:
It seems like we don't need this change ?

Reply:
Ah yep - I think this was left over from before the introduction of the `downloadSeg` package variable. Now that we can stub `downloadSeg` in the tests, this change should no longer be required. Fixed in 75f9d11